ArduPlane/servos.cpp

```diff
@@ -133,11 +133,13 @@ bool Plane::suppress_throttle(void)
         // if we have an airspeed sensor, then check it too, and
         // require 5m/s. This prevents throttle up due to spiky GPS
         // groundspeed with bad GPS reception
+#if AP_AIRSPEED_ENABLED
```

Comment:
this change doesn't look right, it needs an #else

Reply:
Fixed.